project.py

```diff
@@ -443,8 +449,14 @@ def add_arguments(parser):
     parser.add_argument("--skip-clean",
                         help='skip all git and build clean steps before '
                              'building projects',
-                        action='store_true')
-
+                        action='store_true'),
+    parser.add_argument("--build-config",
+                        metavar="NAME",
+                        choices=['debug', 'release'],
+                        dest='build_config',
+                        help='specify "debug" or "release" to override '
+                             'the build configuration in the projects.json file',
+                        default='')
```

Comment:
When a config isn't specified, it shouldn't override the config specified in projects.json. We should just remove the default parameter and let it default to None. There's no need to manually specify an empty string here. I'll make the change and accept this pull request.